.github/release.yml

```diff
@@ -0,0 +1,18 @@
+changelog:
+  exclude:
+    labels:
+      - series/0.22
+      - series/0.23
+  categories:
+    - title: http4s-core
+      labels:
+        - module:http4s-core
+    - title: http4s-server
+      labels:
+        - module:http4s-server
+    - title: http4s-client
+      labels:
+        - module:http4s-client
+    - title: Behind the scenes
+      labels:
+        - "*"
```

Comment:
So apparently `*` means catch-all for anything that wasn't categorized above. Which suggests to me, that e.g. if a PR is labeled for both core and server, it'll get listed under core since that comes first. So, not ideal.

Reply:
I think if it's core and server, labeling it core is fine.  And the modules roughly form a tree, so we can sort them.  The exception would be a few backends that descend from backend-core and (server | client).

A bigger irritant would be anything tagged with siblings: a fix on blaze-server and ember-server.  The list will always require some curation.  We're looking for an 80% solution here.

Reply:
Fair. Anyway we can fiddle with this config at any point. The mergify one is more important to start getting the labels automatically.